Rollout.md

```diff
@@ -193,6 +193,57 @@ I've realized that what I need to do is to first create the order in which the r
 
 ::::::::::::::::::::::::
 
+### Preparation (Feburary)
+
+* We need a documentation site of the different components of the lesson
+  template that uses them. This will serve a dual purpose: (1) actual
+  documentation that early adopters can use to help them understand the new
+  systems; (2) serve as testing ground and a demonstration of how things need to
+  be set up.
+* We need to bring `{sandpaper}` to a usable stage in production (even if some
+  things are broken or not fully implemented).
+* We need to develop a presentation to introduce the new template to the
+  testers.
+* We need to clarify the type of feedback we are interested in getting, how can
+  we measure that these tools are an improvement, and how we are going to
+  collect the feedback. Some [notes in this document](rollout-communication.md).
+
+### Alpha Phase (March)
+
+* Challenge: Engagement on lessons is currently relatively low, which might make
+  testing of these workflows potentially challenging.
+
+#### Group 1: R-based lessons
+
+* 2-3 lessons (one official: R geospatial or R social sciences; one or two from
+  the incubator). They need to be relatively active so the Maintainers will see
+  how the GitHub Actions workflows will work for them.
+* Convert lessons to using `{sandpaper}` with `{pegboard}`, I think at this
+  stage, it might be best if Zhian does it, and based on potential issues
+  encountered, improve and let other Maintainers do it themselves on their
+  lesson in a later testing phase.
```

Comment:
I want to get as much specific feedback as possible, so I think this point we will just test the GitHub workflows on the R maintainers. I fear that if we test both the new template structure with the Pull Request Workflows, we will end up with a situation where something will fail and we will not be sure if it was due to the GitHub workflows or if it was due to the new template structure. Since we are testing the structure on non R-based lessons, we'll be able to get an idea of how that works independently (and get a sense of how difficult it is for non-R users to install R).